### DISCUSSION TITLE
Update capability

### DISCUSSION THREAD

**Discussion by hannahpi** [mechatroner/vscode_rainbow_csv#168] [General]
Hello I see that update seems to be another kind of select statement.  I'm guessing I need to write something to use the output of an update query to "commit" the changes?  or some other tool that merges csv files?  or am I missing something?  I was kind of hoping for a tool that would let me quickly update a csv file that has prices (one price is a single and the other a set).  The RBQL console seems helpful in at least identifying problematic rows for me.  But at this point I'm leaning towards just actually importing my csv file into a database and rewriting the supporting program rather than mess with it.

**Comment by mechatroner** [comment]
The "merge" operation in case of update is very simple - you just need to replace old file content with the content of the result set file. There is also a "Copy Back" button in each result set file at the bottom (in the status bar) that does exactly this. But you can also do it by hand - Select All, Copy, Paste. Rewriting original file is a destructive operation which is why "Update" doesn't do this and creates a new file instead. 

In other words the "update" result set contains non-modified rows too, so there is no need for complex merge.
